lua/colorbox/controller.lua

```diff
@@ -138,6 +140,26 @@ M._parse_args = function(args)
   return opts
 end
 
+M.shuffle = function()
+  local ColorNamesList = runtime.colornames()
+  if #ColorNamesList > 0 then
+    local logger = logging.get("colorbox")
+    local previous_track = track.previous_track()
+    local i = previous_track ~= nil and previous_track.color_number or 0
+    local color = track.get_next_color_name_by_idx(i)
```

Comment:
Hi @marcelarie,

I mean in this line, it doesn't look like to do shuffle/random selection.

Please see: https://github.com/linrongbin16/colorbox.nvim/blob/c95d361c7d1387a677ea9968cbc55ca0cfb62500/lua/colorbox/policy/builtin.lua#L13

Reply:
Now it should be working correctly [62a6dab](https://github.com/linrongbin16/colorbox.nvim/pull/239/commits/62a6dabb8e3415c8363828d56e5f5b02a04471ad)